tst/EnergyPlus/unit/HVACControllers.unit.cc

```diff
@@ -233,12 +233,6 @@ TEST_F(EnergyPlusFixture, HVACControllers_TestTempAndHumidityRatioCtrlVarType)
     state->dataAirLoop->AirToZoneNodeInfo(1).ZoneEquipSupplyNodeNum.allocate(1);
     state->dataAirLoop->AirToZoneNodeInfo(1).ZoneEquipSupplyNodeNum(1) = 4;
     state->dataConvergeParams->AirLoopConvergence.allocate(1);
-    state->dataConvergeParams->AirLoopConvergence(1).HVACMassFlowNotConverged.allocate(2);
-    state->dataConvergeParams->AirLoopConvergence(1).HVACHumRatNotConverged.allocate(2);
-    state->dataConvergeParams->AirLoopConvergence(1).HVACTempNotConverged.allocate(2);
-    state->dataConvergeParams->AirLoopConvergence(1).HVACEnergyNotConverged.allocate(2);
-    state->dataConvergeParams->AirLoopConvergence(1).HVACEnthalpyNotConverged.allocate(2);
-    state->dataConvergeParams->AirLoopConvergence(1).HVACPressureNotConverged.allocate(2);
```

Comment:
No more needing to allocate these, :+1:

Reply:
Every time a heap object is converted to an automatic inline object, an angel gets its wings.

Reply:
:angel: :angel: :angel: :angel: :angel: